scripts/drf_tests_settings.py

```diff
@@ -1,7 +1,7 @@
 SECRET_KEY = "1"
 SITE_ID = 1
 
-INSTALLED_APPS = [
+INSTALLED_APPS = (
     "django.contrib.contenttypes",
     "django.contrib.sites",
```

Comment:
INSTALLED_APPS must be tuple for typecheck_tests, because DRF test suite does:
```python
settings.INSTALLED_APPS += ('guardian',)
```

Better than ignoring IMO.